agent/Makefile

```diff
@@ -250,10 +250,13 @@ install-lib:
 	${INSTALL} ${INSTALLOPTS} ${LIBDIR}
 	${COPY} lib ${DESTDIR}
 
+# NOTE: we deliberately use `/usr/bin/python3` to make sure that when this is
+# invoked inside a python virtualenv the pip installation won't uninstall the
+# existing pbench module, breaking the environment.
```

Comment:
My head hurts in sympathy for your pain ... 😵‍💫 

But more practically ... is this going to be an issue if we try to expand our `tox` to work with multiple Python environments, since we're always forcing the install through the "default" Python?

Reply:
I don't know.  I suspect we need to run that install by either having it detect that it is in a CI job and using the artifacts of the virtualenv of the CI job, and otherwise using the available `python3` interpreter out side our CI environment.

Reply:
In other words, we don't know, but this is a step forward. Got it. We really ought to test in multiple environments unless we can actually standardize support on a single environment (which I doubt will be practical any time soon); but boiling the ocean to make a cup of tea is probably a bad idea so "carry on".